pyasdf/asdf_data_set.py

```diff
@@ -1833,7 +1833,7 @@ def process(self, process_function, output_filename, tag_map,
         if self.mpi:
             self.mpi.comm.barrier()
 
-        if not self.mpi and self.__compression[0] is not None:
+        if not self.mpi and self.__compression[0]:
```

Comment:
I think this can now be just `if not self.mpi:` as `self.__compression[0]` is now always set to something hopefully reasonable.

Reply:
I fixed this in the latest commit